utilities/verify.sh

```diff
@@ -10,7 +10,7 @@ if [ "${TRAVIS_PULL_REQUEST}" == "false" ]; then
     chmod 700 $TRAVIS_BUILD_DIR/signing-tools
     tar xvf $TRAVIS_BUILD_DIR/signing-tools.tar -C $TRAVIS_BUILD_DIR/signing-tools
     # Run verify
-    mvn verify
+    mvn verify -P release
```

Comment:
Ok, I am not sure what kind of problem can be caused in the packaging stage (I assumed this is plain jarring and also was not sure that we care to do that on each platform) but I get your point. Fine as is.